packages/env/lib/config.js

```diff
@@ -286,6 +285,32 @@ function getNumberFromEnvVariable( varName ) {
 	return maybeNumber;
 }
 
+/**
+ * Gets the `wp-env` home directory in which generated files are created.
+ *
+ * By default, '~/.wp-env/'. On Linux, '~/wp-env/'. Can be overriden with the
+ * WP_ENV_HOME environment variable.
+ *
+ * @return {string} The absolute path to the `wp-env` home directory.
+ */
+function getWpEnvHomeDir() {
```

Comment:
I'd suggest renaming to `getHomeDirectory()`. `WpEnv` is redundant because all of this code refers to `wp-env`, and it's best to avoid abbreviations where we can.